pkg/webhook/route/defaults.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2021 Clastix Labs
+// SPDX-License-Identifier: Apache-2.0
+
+package route
+
+import (
+	capsulewebhook "github.com/clastix/capsule/pkg/webhook"
+)
+
+// +kubebuilder:webhook:path=/defaults,mutating=true,sideEffects=None,admissionReviewVersions=v1,failurePolicy=fail,groups="",resources=pods,verbs=create,versions=v1,name=pod.defaults.capsule.clastix.io
+// +kubebuilder:webhook:path=/defaults,mutating=true,sideEffects=None,admissionReviewVersions=v1,failurePolicy=fail,groups="",resources=persistentvolumeclaims,verbs=create,versions=v1,name=storage.defaults.capsule.clastix.io
+// +kubebuilder:webhook:path=/defaults,mutating=true,sideEffects=None,admissionReviewVersions=v1,failurePolicy=fail,groups=networking.k8s.io,resources=ingresses,verbs=create;update,versions=v1beta1;v1,name=ingress.defaults.capsule.clastix.io
```

Comment:
We can use a single annotation and separate groups with the `;` symbol: this would lead to a single new webhook in the `mutatingwebhookconfiguration` resource and reduce the possible human error when aligning the Kustomize and the Helm chart.

Reply:
But how do you want to deal with the fact, that only ingresses use the `update` verb, but the others don't?

Reply:
Thanks for pointing that, you're right. Then, let's keep 3 different rules for that.